server/src/database/MainDatabase.hh

```diff
@@ -0,0 +1,131 @@
+#pragma once
+
+#include <oatpp/orm/DbClient.hpp>
+#include <oatpp/core/macro/codegen.hpp>
+#include <oatpp/orm/SchemaMigration.hpp>
+#include <filesystem>
+#include <ranges>
+#include <vector>
+
+#include "dto/UserDto.hh"
+#include "dto/PlaceDto.hh"
+#include "dto/OfferDto.hh"
+
+#include OATPP_CODEGEN_BEGIN(DbClient)
+
+namespace server::database {
+
+    class MainDatabase : public oatpp::orm::DbClient {
+    public:
+        [[nodiscard]]
+        explicit
+        MainDatabase(std::shared_ptr<oatpp::orm::Executor> const &executor)
```

Comment:
The constructor should be moved to .cc file.

Reply:
Also shared_ptr should be passed by value not reference.

Reply:
We don't have a `.cc` file for this class. Should I create it?

Reply:
Yeah